README.md

```diff
@@ -22,9 +22,13 @@ This action integrates asana with github.
 
 **Optional** If any comment is provided, the action will add a comment to the specified asana task with the text & pull request link.
 
+### `target-project`
+
+**Optional** Move task only if it exists in provided project i.e `Current Sprint`.
```

Comment:
I don't think this is optional, unless I'm mis-reading the code below. `if (sectionName && projectName) {` makes it looks like nothing will happen if either `target-section` or `target-project` are omitted.

Reply:
`target-project` and `target-section` are required to move the task from one section to another. If we only want to comment PR link on task and don't want to move the task to a specific section then both of these parameters can be omitted. `if (taskComment) {` is outside of `if (sectionName && projectName) {` block.